.editorconfig

```diff
@@ -243,12 +243,30 @@ dotnet_diagnostic.IDE0004.severity = error
 # IDE0019: Use pattern matching
 csharp_style_pattern_matching_over_as_with_null_check = true:error
 
+# IDE0130: Warn and provide code fixes for when namespaces do not match the folder structure.
+dotnet_diagnostic.IDE0130.severity = warning
```

Comment:
This one adds a few warnings but it seems reasonable and MSVS allows to fix the issues automatically using a code fix.

Reply:
I think this is useful.